SubEthaEdit-Mac/Source/TCMBEEP/TCMBEEPSession.m

```diff
@@ -142,14 +142,13 @@ + (void)setAnonCiphersOnStreamData:(CFDataRef)inSocketDataRef dhParams:(BOOL)aFl
 	}
 }
 
-- (void)TCM_initHelper
-{
+- (void)TCM_initHelper {
 	self.uniqueID = [NSString UUIDString];
     I_flags.hasSentTLSProceed = NO;
     I_flags.isWaitingForTLSProceed = NO;
     I_flags.isTLSHandshaking = NO;
     I_flags.isTLSEnabled = NO;
-    CFStreamClientContext context = {0, (__bridge void * _Null_unspecified)(self), NULL, NULL, NULL};
+    CFStreamClientContext context = {0, (__bridge void *)(self), NULL, NULL, NULL};
```

Comment:
lets keep the nullability annotations out for now, I don't see much value in them anyways, and if youre going all in, warnings galore is happening for you too soonish.